src/gridfs.rs

```diff
@@ -221,7 +228,7 @@ impl GridFsBucket {
         Ok(())
     }
 
-    /// Finds and returns the [`FilesCollectionDocument`]s within this bucket that match the given
+    /// Finds and returns the [`FilesCollectionDocument`]s within this bucket that matches the given
```

Comment:
Suggest reverting, `FilesCollectionDocument`s is plural here
```suggestion
    /// Finds and returns the [`FilesCollectionDocument`]s within this bucket that match the given
```